sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -257,7 +261,12 @@ def from_connection_string(
         :keyword dict http_proxy: HTTP proxy settings. This must be a dictionary with the following
          keys: `'proxy_hostname'` (str value) and `'proxy_port'` (int value).
          Additionally the following keys may also be present: `'username', 'password'`.
-        :rtype: ~azure.servicebus.ServiceBusReceiverClient
+        :keyword bool is_dead_letter_receiver: Should this receiver connect to the dead-letter-queue associated
```

Comment:
Is this public?

Reply:
yes.. this is public.
Although most user would be using `ServiceBusClient` to get a dead letter receiver, we still provides users the ability to directly create a `ServiceBusReceiver` -- then we probably want to allow users to create a dead letter receiver directly.

Do you have any concern on this? -- naming, or it shouldn't be exposed at all?

Reply:
See my earlier comment in the populate_ function, it seems relevant to this discussion :)

(Tersely, I'm not convinced we should have pulled in ALL of the variables into populate_, in my proposal I had only brought in prefetch and some others that were valid for all receivers; I see these vars more as mutations on the entity_path/URI that we pass to the generic shared receiver logic, and should thus be "worked out" earlier in the flow)

Reply:
as per discussion with Kieran OOB, decided to remove "is_dead_letter_receiver" and "transfer_deadletter" from the surface of `ServiceBusReceiver` for now so that we don't get our hands tied in future decisions.

This could be a post GA feature whether to provide helper methods to generate DLQ path or adding options in `ServiceBusReceiver` constructor.

Reply:
Sounds good!